tests/bettertransformer/test_bettertransformer_encoder.py

```diff
@@ -49,6 +49,7 @@
 ALL_ENCODER_DECODER_MODELS_TO_TEST = [
     "hf-internal-testing/tiny-random-FSMTModel",
     "hf-internal-testing/tiny-random-BartModel",
+    "hf-internal-testing/tiny-random-TapasModel",
```

Comment:
Could you move this to `ALL_ENCODER_MODELS_TO_TEST` instead? `Tapas` is an encoder-only model ;)

Reply:
Reading https://huggingface.co/docs/transformers/v4.24.0/en/model_doc/tapas#transformers.TapasModel

> The model can behave as an encoder (with only self-attention) as well as a decoder, in which case a layer of cross-attention is added between the self-attention layers, following the architecture described in [Attention is all you need](https://arxiv.org/abs/1706.03762) by Ashish Vaswani, Noam Shazeer, Niki Parmar, Jakob Uszkoreit, Llion Jones, Aidan N. Gomez, Lukasz Kaiser and Illia Polosukhin.

Reply:
Oh I see! Thanks for an insightful explanation!  I just moved it! :)